routes/index.js

```diff
@@ -20,4 +22,8 @@ apiRouter.use((error, req, res, next) => {
     res.send(error);
 });
 
+apiRouter.use((error, req, res, next) => {
+    res.send(error);
+});
+
```

Comment:
It would be great if we could add a 404 (Page Not Found) handler above this one: 
```
// Create custom 404 handler that sets the status code to 404.
server.use('*', (req, res, next) =>{
  res.status(404);
  res.send({ error: 'route not found'});
})
```

Reply:
updated and resolved.